components/web/squid/Makefile

```diff
@@ -22,45 +22,45 @@
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
 # Copyright (c) 2019, Michal Nowak
+# Copyright (c) 2019, Friedrich Kink
 #
 
+BUILD_BITS=             64
+USE_OPENSSL11=		yes
 include ../../../make-rules/shared-macros.mk
 
-PATH=/usr/bin:/usr/gnu/bin:/usr/sbin:/usr/perl5/bin
-
 COMPONENT_NAME=         squid
-COMPONENT_VERSION=      3.5.28
-COMPONENT_SUMMARY=	Squid Web Proxy Cache
+COMPONENT_VERSION=      5.3
+COMPONENT_REVISION=     1
+COMPONENT_SUMMARY=      Squid Web Proxy Cache
```

Comment:
COMPONENT_REVISION needs to be reset as you have changed the COMPONENT_VERSION and we didn't have released it yet.